### ISSUE TITLE
[sklearn#34117] Sparse coefficients in linear models are not documented

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#352]
**Mirrored from [scikit-learn#34117](https://github.com/scikit-learn/scikit-learn/issues/34117)**

Original title: Sparse coefficients in linear models are not documented
Original labels: Documentation
Created: 2026-05-25T11:20:04Z

---

### Describe the issue linked to the documentation

As a follow up from https://github.com/scikit-learn/scikit-learn/pull/34093

There are several classes of linear models in scikit-learn that can have sparse coefficients, but the documentation (other than for LogisticRegression which was updated in the PR above) mentions that the coefficients are `ndarray`.

### Suggest a potential alternative/fix

All classes of linear models with sparsifyable coefficients should mention in their documentation that their attribute `coef_` can be either ndarray or sparse CSR.

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Addressed in upstream PR(s). See scikit-learn/scikit-learn PRs for #34117.
